src/unittest/test_serialization.cpp

```diff
@@ -286,16 +284,15 @@ void TestSerialization::testStreamWrite()
 	writeV2S32(os, v2s32(1920, 1080));
 	writeV3S32(os, v3s32(-400, 6400054, 290549855));
 
-	os << serializeWideString(L"\x02~woof~\x5455");
-
 	writeV3F1000(os, v3f(500, 10024.2f, -192.54f));
 	writeARGB8(os, video::SColor(255, 128, 50, 128));
 
-	os << serializeLongString("some longer string here");
+	os << serializeString32("some longer string here");
 
 	writeU16(os, 0xF00D);
 
 	data = os.str();
+	std::cout << data.size() << " & " << sizeof(test_serialized_data) << std::endl;
```

Comment:
should be removed probably?

Reply:
Oops! Yes sure, this was a left-over from fixing the unittests.